.gitignore

```diff
@@ -6,3 +6,4 @@ logs
 *.iml
 .idea
 .DS_Store
+/bin
```

Comment:
What was this added?

Reply:
I'm not sure, I didn't add it explicitly. I had some trouble importing it into Eclipse, and I think it generated a bin folder and then added that line so it wouldn't push to github. I deleted the bin folder and that line form the file and it compiles fine and still passes all the tests.

Reply:
Did you import as a maven project?  This is probably the best way to go w/ eclipse.

Reply:
nope I just did plain Java project, that must have been why it was giving me so much trouble. Just ended up using sublime instead.

Reply:
Importing a maven project is very nice because it uses pom to automatically setup dependencies.  Also it will build to target dir instead of bin.